samples/net/wifi/apsta_mode/boards/frdm_rw612.conf

```diff
@@ -4,6 +4,9 @@ CONFIG_ETH_DRIVER=n
 CONFIG_WIFI_NM=y
 CONFIG_NXP_WIFI_SOFTAP_SUPPORT=y
 
+CONFIG_WIFI_AP_IP_ADDRESS="192.168.10.1"
```

Comment:
This cannot be here as now someone having this board will get this IP address by default which is probably not something user wants.
Also the DHCPv4 option should be set somewhere else that needs to disable the setting.